{
  "contest": 193,
  "handle": "dragotanqueray",
  "risk": "2",
  "title": "The create() function does not catch issues with arguments' validity",
  "issueId": 105,
  "issueUrl": "https://github.com/code-423n4/2022-12-caviar-findings/issues/105"
}